gravatar-ui/src/main/java/com/gravatar/ui/components/atomic/ExpandableText.kt

```diff
@@ -37,25 +46,54 @@ internal fun ExpandableText(text: String, modifier: Modifier = Modifier, maxLine
     )
     if (showDialog) {
         Dialog(onDismissRequest = { showDialog = false }) {
-            Card(
-                modifier = Modifier.wrapContentSize(),
-                shape = RoundedCornerShape(16.dp),
-            ) {
-                Text(
-                    text = text,
-                    modifier = Modifier
-                        .padding(16.dp)
-                        .fillMaxWidth()
-                        .wrapContentSize(),
-                    textAlign = TextAlign.Start,
-                )
-            }
+            dialogContent?.let { dialogContent(text) }
```

Comment:
No need to update that but TIL we can write ` dialogContent?.invoke(text)`

Reply:
🤦🏼‍♂️ thanks!